sdk/cosmos/azure-cosmos-spark_3_2-12/Samples/Scala/filteringDuringMigration.scala

```diff
@@ -0,0 +1,159 @@
+// Databricks notebook source
+val cosmosEndpoint = "<cosmos-endpoint>"
+val cosmosMasterKey = "<cosmos-master-key>"
+val cosmosDatabaseName = "ContosoHospital"
+val cosmosContainerName = "Patient"
+
+
+// Patient Document -- partition key: patientId
+// {
+//   "id": "9c9a1156-e936-40f3-a442-e9528b55a2fb",
+//   "patientId": "423ab2cf-dd1c-4404-8524-86cee045c179",
+//   "patientName": "John Doe",
+//   "doctorId" : "629f49da-9cfc-45a4-8e1c-d4f8b7ab1f4e",
+//   "doctorName": "Sung Ondricka"
+// }
+
+
+//-----filtering examples-with-schema-inference-disabled---------------------
+
+val TargetContainerName = "CopyContainer"
+val checkpointLocation = "/tmp/streaming_checkpoint"
+
+val changeFeedCfg = Map("spark.cosmos.accountEndpoint" -> cosmosEndpoint,
```

Comment:
Usually customers would want throughput control being enabled on the source (to avoid side effects of the migration to the source environment). Can you please add the throughput control config as well - at least add commented settings and instruction on how to enable i if the customers wants to use it?

Reply:
Thanks.